protocol/storage/storagedeal/types.go

```diff
@@ -124,11 +127,15 @@ type Deal struct {
 }
 
 // ProofInfo contains the details about a seal proof, that the client needs to know to verify that his deal was posted on chain.
-// TODO: finalize parameters
 type ProofInfo struct {
+	// Sector id allows us to find the committed sector metadata on chain
 	SectorID uint64
-	CommR    []byte
-	CommD    []byte
+
+	// CommitmentMessage is the cid of the message that committed the sector. It's used to track when the sector goes on chain.
+	CommitmentMessage *cid.Cid
+
+	// PieceInclusionProof is a proof that a the piece is included within a sector
+	PieceInclusionProof []byte
```

Comment:
Could you justify for me removing the CommR and CommD?

For CommD, I understand it's part of the PIP, so that makes sense, but CommR isn't. Even if it's in the CommitmentMessage, that's not sufficient to gaurantee the client can access it given the message CID, at least until it's mined into a block. Since this message is not on-chain data, I'd suggest leaving CommR here to simplify the client's bookkeeping.

Reply:
> Could you justify for me removing the CommR and CommD?

If the client has the content id of the `commitSector` message and the `commitSector` message includes data and replica commitments, then the replica and data commitments are redundant (in this data structure).

Reply:
@anorth CommR is used to verify the sector is correctly sealed. The client can't use it.
At a higher level, any information about the sector (other than its id) that the miner delivers here is kind of worthless because they haven't proven anything. The appropriate way for a client to access CommD and CommR is to look it up on their local chain by sector id via a query message.

Since we already have `verifyPieceInclusion` implemented in `miner.Actor`, clients won't need `CommD` at all. They can just call `verifyPieceInclusion` with their `CommP`, the miners piece inclusion proof, and the `sectorID`.